sklearn/manifold/t_sne.py

```diff
@@ -672,6 +676,12 @@ def __init__(self, n_components=2, perplexity=30.0,
         self.method = method
         self.angle = angle
 
+        # TODO sklearn 0.20: remove this warning
+        if learning_rate == 200.0:
```

Comment:
Let's try to find consensus on the default value change before we decide on how we want the warnings to happen. See https://github.com/scikit-learn/scikit-learn/pull/8397#issuecomment-281628367.